client/components/Location.jsx

```diff
@@ -1,24 +1,24 @@
 import React from 'react';
```

Comment:
Please revert changes back to original. (I have similar fixes on my current branch.)